src/test/java/io/openliberty/tools/common/plugins/util/BaseDevUtilTest.java

```diff
@@ -147,6 +147,11 @@ public void runUnitTests() throws PluginScenarioException, PluginExecutionExcept
         public void runIntegrationTests() throws PluginScenarioException, PluginExecutionException {
             // not needed for tests
         }
+
+        @Override
+        public void redeployApp() throws Exception {
```

Comment:
should we throws explicit exception, such as PluginExecutionException, so that we can catch it in the recompileJava() method and output the meaning error message.

Reply:
Fixed with https://github.com/OpenLiberty/ci.common/pull/94/commits/3d1a1238bd465a59602014dc4a8f51e27c5bd0f2